tests/integration/helpers.py

```diff
@@ -820,6 +821,22 @@ async def switchover(
             assert standbys == len(ops_test.model.applications[app_name].units) - 1
 
 
+async def switchover_to_unit_zero(ops_test: OpsTest) -> None:
```

Comment:
Used to reduce the number of switchovers to the minimum (as we expect since the previous refresh/upgrade implementation).